v1/content.yaml

```diff
@@ -126,6 +126,14 @@ paths:
           description: The revision id
           type: integer
           required: true
+      x-request-handler:
+        - get_from_backend:
+            request:
+              uri: /{domain}/sys/page_revisions/page/{title}/{revision}
```

Comment:
Why are you replicating this here? swagger-router should know to use `{revision}` only if it's present, so the URI can be the same for both cases.

(This question goes for all replications in the files below)

Reply:
> swagger-router should know to use {revision} only if it's present, so the URI can be the same for both cases

Tested it a bit and it doesn't seem to always be the case. Reverted it back to the original using optional params since it doesn't affect the final swagger spec.

Reply:
Made 2 other commits testing the link in question and they both failed some tests. There was a at least one that didn't earlier today. Will try to recreate tomorrow.

Reply:
Hmmm this is a bit weird, it shouldn't be failing at all with `{/revision}` (as swagger-router knows not to attach the slash and the rev param if it's not there).

Reply:
Maybe some miscommunication here. It does not fail with `{/revision}` that's why I reverted it after your comment. It fails with `/{revision}`

Reply:
We actually need to get rid of the optional syntax completely if we want to be fully v3-compliant, so yes, copy/pasta and duplication are needed here. Thnx @clarakosi for pointing out the obvious.